src/backend/protocols/go.twitter/worker.go

```diff
@@ -102,7 +119,7 @@ func InitAndStartWorker(conf WorkerConfig) (worker *Worker, err error) {
 
 	worker.NatsConn, err = nats.Connect(conf.BrokerConfig.NatsURL)
 	if err != nil {
-		log.WithError(err).Warn("[EmailBroker] initalization of NATS connexion failed")
+		log.WithError(err).Warn("[TwitterBroker] initalization of NATS connexion failed")
```

Comment:
error not a warning

Reply:
don't know. error is not returned and function continue its process, so for me its a warning